pkg/edit/editor_test.go

```diff
@@ -37,6 +37,36 @@ func TestEditor_DoesCommandWithLeadingSpaceToHistory(t *testing.T) {
 	testCommands(t, f.Store /* no commands */)
 }
 
+func TestEditor_TestBeforeAddCmd(t *testing.T) {
+	cases := []struct {
+		name string
+		rc   string
+		in   string
+		exp  []string
+	}{
+		{"nonCallableItem", "edit:add-cmd-filters = [$false]", "echo\n", []string{"echo"}},
+		{"noOutputItem", "edit:add-cmd-filters = [[_]{}]", "echo\n", []string{"echo"}},
```

Comment:
Since these will pollute the terminal when run, I'd rather you comment them out now and reenable them later.